src/ClientData/FastRenderingUtilsTest.cpp

```diff
@@ -8,8 +8,8 @@
 
 namespace orbit_client_data {
 
-constexpr uint64_t kStartNs = 100;
-constexpr uint64_t kEndNs = 200;
+constexpr uint64_t kStartNs = 100'000'000'000'000ULL;
+constexpr uint64_t kEndNs = 200'000'000'000'000ULL;
```

Comment:
In a way this changes a couple of the tests below quite a bit, as it changes `kEndNs - kStartNs` from 100 to 10^11. Is that ok?

Reply:
Yes, It's actually better.